library/entropy_poll.c

```diff
@@ -18,9 +18,14 @@
  */
 
 #if defined(__linux__)
+#include <features.h>
```

Comment:
Question: is it OK to include that header before we define `_GNU_SOURCE`? I thought feature macros such as `_GNU_SOURCE` had to be defined before including any header, but perhaps that one is an exception?

Reply:
`features.h` is for `__GLIBC__ ` macro.
`_GNU_SOURCE` should be at first.
I will fix it.